diff --git a/node_modules/resume-cli/build/export-resume.js b/node_modules/resume-cli/build/export-resume.js
index e023577..f15157b 100644
--- a/node_modules/resume-cli/build/export-resume.js
+++ b/node_modules/resume-cli/build/export-resume.js
@@ -111,7 +111,8 @@ const createPdf = (resumeJson, fileName, theme, format, callback) => {
       themePath: theme
     });
     const browser = await puppeteer.launch({
-      args: puppeteerLaunchArgs
+      args: puppeteerLaunchArgs,
+      headless: 'new'
     });
     const page = await browser.newPage();
     await page.emulateMediaType(themePkg.pdfRenderOptions && themePkg.pdfRenderOptions.mediaType || 'screen');
